app/controllers/users/piv_cac_recommended_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Users
+  class PivCacRecommendedController < ApplicationController
+    include TwoFactorAuthenticatableMethods
+    include MfaSetupConcern
+    include SecureHeadersConcern
+    include ReauthenticationRequiredConcern
+
+    before_action :authenticate_user!
+    before_action :confirm_user_authenticated_for_2fa_setup
+    before_action :apply_secure_headers_override
+    before_action :user_email_is_gov_or_mil?
+
+    helper_method :in_multi_mfa_selection_flow?
+
+    def show
+      @email_type = email_type
+      @skip_text = skip_text
+      analytics.piv_cac_recommended_page_visited
+    end
+
+    def confirm
+      UpdateUser.new(user: current_user, attributes: { piv_cac_recommended_dismissed: true }).call
+      user_session[:mfa_selections] = ['piv_cac']
+      analytics.piv_cac_recommended_accepted
```

Comment:
Maybe we can have a single "submitted" event with accepted/skipped as a property? Could make it easier to chart analytics for comparing selections.

Reply:
ooo good idea